examples/isolated-functions/isolated_functions.description

```diff
@@ -0,0 +1,4 @@
+// An `isolated` function is a function that accesses mutable state only via its parameters.
```

Comment:
```suggestion
// An `isolated` function is a function that accesses a mutable state only via its parameters.
```

Reply:
Resolving these as per the offline discussion.